pyclient/pydeephaven/_console_service.py

```diff
@@ -0,0 +1,73 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+
+from pydeephaven.dherror import DHError
+from pydeephaven.proto import console_pb2_grpc, console_pb2
+from pydeephaven.table import Table
+
+
+class ConsoleService:
```

Comment:
check with https://github.com/deephaven/deephaven-core/pull/1082 as this might change some of your usage here

Reply:
thanks for the heads up